src/Tabs/Tab.js

```diff
@@ -182,7 +182,7 @@ class Tab extends React.Component {
 
     let style = {};
 
-    if (textColor !== 'secondary' && textColor !== 'inherit') {
+    if (textColor !== 'primary' && textColor !== 'secondary' && textColor !== 'inherit') {
       style.color = selected ? textColor : theme.palette.text.secondary;
```

Comment:
@oliviertassinari This ability use an arbitrary color value in addition to the theme defined ones is an interesting deviation from the standard pattern. What do you think about it?